client/my-sites/pages/page/index.js

```diff
@@ -605,6 +635,7 @@ const mapDispatch = {
 	recordMoreOptions: partial( recordEvent, 'Clicked More Options Menu' ),
 	recordPageTitle: partial( recordEvent, 'Clicked Page Title' ),
 	recordEditPage: partial( recordEvent, 'Clicked Edit Page' ),
+	recordFrontPage: partial( recordEvent, 'Clicked Set as Front Page' ),
```

Comment:
As noted above, we can just remove this line. It's for tracking a Google Analytics event, which we should enable along with the rest of the feature.